app/frontend/src/components/settings/lists/StackedListIcon.js

```diff
@@ -0,0 +1,44 @@
+import { Container, Image } from 'react-bootstrap';
+
+function StackedListIcon(props){
+    return(
+        <>

```

Comment:
Lets not use css in a <Style> block in here. In this case especially it would be included in the html for each rendered list preview.
Consider creating a seperate .css file.